libs/template/helpers_test.go

```diff
@@ -32,6 +32,24 @@ func TestTemplatePrintStringWithoutProcessing(t *testing.T) {
 	assert.Equal(t, `{{ fail "abc" }}`, cleanContent)
 }
 
+func TestTemplateBundleUuidFunction(t *testing.T) {
+	ctx := context.Background()
+
+	ctx = root.SetWorkspaceClient(ctx, nil)
+	helpers := loadHelpers(ctx)
+	r, err := newRenderer(ctx, nil, helpers, os.DirFS("."), "./testdata/bundle-uuid/template", "./testdata/bundle-uuid/library")
+	require.NoError(t, err)
+
+	err = r.walk()
+	assert.NoError(t, err)
+
+	assert.Len(t, r.files, 2)
+	c1 := strings.Trim(string(r.files[0].(*inMemoryFile).content), "\n\r")
+	assert.Equal(t, strings.Repeat(bundleUuid, 3), c1)
+	c2 := strings.Trim(string(r.files[1].(*inMemoryFile).content), "\n\r")
+	assert.Equal(t, strings.Repeat(bundleUuid, 5), c2)
```

Comment:
Why assert on multiple files? If there is a specific reason for this please include a comment.